Update lisk-bft to use lisk-dpos to require as module

### Expected behavior
Should use `liskhq/lisk-dpos` library to require.
### Actual behavior
```
FAIL test/bft_fork_choice_rules_protocol_specs.spec.js
@liskhq/lisk-bft:   ● Test suite failed to run
@liskhq/lisk-bft:     Cannot find module '../../../../../../../src/modules/chain/dpos' from 'bft_fork_choice_rules_protocol_specs.spec.js'
@liskhq/lisk-bft:       16 | 
@liskhq/lisk-bft:       17 | const forkChoiceSpecs = require('./bft_specs/bft_fork_choice_rules.json');
@liskhq/lisk-bft:     > 18 | const { Slots } = require('../../../../../../../src/modules/chain/dpos');
@liskhq/lisk-bft:          |                   ^
@liskhq/lisk-bft:       19 | const { BFT } = require('../../../../../../../src/modules/chain/bft');
@liskhq/lisk-bft:       20 | const { constants } = require('../../../../../../utils');
@liskhq/lisk-bft:       21 | 
@liskhq/lisk-bft:       at Resolver.resolveModule (node_modules/jest/node_modules/jest-resolve/build/index.js:259:17)
@liskhq/lisk-bft:       at Object.<anonymous> (test/bft_fork_choice_rules_protocol_specs.spec.js:18:19)
@liskhq/lisk-bft: PASS test/fork_choice_rule.spec.js
```
### Steps to reproduce
- `npm run test` in bft module
### Which version(s) does this affect? (Environment, OS, etc...)
development

--- a/elements/lisk-bft/test/bft_fork_choice_rules_protocol_specs.spec.js
+++ b/elements/lisk-bft/test/bft_fork_choice_rules_protocol_specs.spec.js
@@ -15,9 +15,14 @@
 'use strict';
 
 const forkChoiceSpecs = require('./bft_specs/bft_fork_choice_rules.json');
-const { Slots } = require('../../../../../../../src/modules/chain/dpos');
-const { BFT } = require('../../../../../../../src/modules/chain/bft');
-const { constants } = require('../../../../../../utils');
+const { Slots } = require('@liskhq/lisk-dpos');
+const { BFT } = require('../src/bft');
+
+const constants = {
+	ACTIVE_DELEGATES: 101,
+	EPOCH_TIME: '2016-05-24T17:00:00.000Z',
+	BLOCK_TIME: 10,
+};
 
 describe('bft', () => {
 	describe('forkChoice', () => {

